VCTwoFluidStaggeredStokesOperator.cpp

```diff
@@ -60,12 +60,10 @@ static const int SIDEG = 1;
 
 // Types of refining and coarsening to perform prior to setting coarse-fine
 // boundary and physical boundary ghost cell values.
-static const std::string CC_DATA_REFINE_TYPE =
-    "CONSERVATIVE_LINEAR_REFINE"; // how to fill in fine cells from coarse cells, how to fill ghost cells on refine
-                                  // patch
-static const std::string SC_DATA_REFINE_TYPE =
-    "CONSERVATIVE_LINEAR_REFINE";              // how to fill in fine cells from coarse cells, how to fill ghost
-                                               // cells on refine patch
+static const std::string CC_DATA_REFINE_TYPE = "NONE"; // how to fill in fine cells from coarse cells, how to fill ghost
+                                                       // cells on refine patch
+static const std::string SC_DATA_REFINE_TYPE = "NONE"; // how to fill in fine cells from coarse cells, how to fill ghost
+                                                       // cells on refine patch

```

Comment:
Why do we set these to none? I thought we wanted to fill in ghost cells near the coarse-fine interfaces using conservative_linear_refine.

Reply:
The answer is in the next line. The refine operators fill in ghost cells from coarse data. We want to use coarse-fine interpolation to fill ghost cells which uses data from coarse and fine cells.

 I think that changing these from `"NONE"` to `"CONSERVATIVE_LINEAR_REFINE"` does nothing as long as `USE_CF_INTERPOLATION = true`. You can check that.

However, I don't think `USE_CF_INTERPOLATION = true` will fill in corner ghost cells, so if those are required, then using `"CONSERVATIVE_LINEAR_REFINE"` is needed.